src/NHibernate.Test/DebugSessionFactory.cs

```diff
@@ -280,7 +281,11 @@ ISession ISessionFactory.GetCurrentSession()
 
 		ITransactionFactory ISessionFactoryImplementor.TransactionFactory => ActualFactory.TransactionFactory;
 
-		UpdateTimestampsCache ISessionFactoryImplementor.UpdateTimestampsCache => ActualFactory.UpdateTimestampsCache;
+		public UpdateTimestampsCache UpdateTimestampsCache
+		{
+			get => _updateTimestampsCache ?? ActualFactory.UpdateTimestampsCache;
+			set { _updateTimestampsCache = value; }
```

Comment:
Please use consistent code style. Either both need to be in brackets or to be lambdas.

Reply:
Ok!